apps/www/components/glossary/terms-navigation-desktop.tsx

```
@@ -0,0 +1,113 @@
+        </Button>
+        <div className="overflow-hidden flex-grow">
+          {visibleTerms.map((term) => (
+            <Link
+              href={`/glossary/${term.slug}`}
+              className={cn(
+                "flex items-center px-2 h-10 rounded-md text-white/60 hover:text-white",
+                {
+                  "text-white": slug === term.slug,
+                },
+              )}
+            >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)